avocado/utils/configure_network.py

```diff
@@ -201,3 +202,18 @@ def is_interface_link_up(interface):
         log.info("Interface %s link is up", interface)
         return True
     return False
+
+
+def get_interface_ip(interface):
```

Comment:
Maybe a better name? Is this returning the ipv4 or ipv6 address? Looks like is ipv4, right?

Suggestion: `get_ipv4_address()`

Reply:
@beraldoleal 
Oh, you are right
Maybe, I can add param which IP version it should be revert.
By default to get ipv4.
How do you think?

Reply:
@anascko It is a plan. Or we could have two different methods and a third-one with your parameter, like:

```python
def get_ip_address(interface, version=4):
   if version == 4:
      return self.get_ipv4_address(interface)
   elif version == 6:
     return self.get_ipv6_address(interface)
   else:
     raise Exception("Version not supported")
```

What do you think?